src/sentry/demo/tasks.py

```diff
@@ -0,0 +1,40 @@
+from datetime import timedelta
+from django.conf import settings
+from django.db.models import F
+from django.utils import timezone
+
+from sentry.models import Organization, OrganizationStatus, User
+from sentry.tasks.base import instrumented_task
+from sentry.tasks.deletion import delete_organization
+
+
+MAX_RETRIES = 5
+
+
+@instrumented_task(
+    name="sentry.demo.tasks.delete_users_orgs",
+    default_retry_delay=60 * 5,
+    max_retries=MAX_RETRIES,
+)
+def delete_users_orgs(**kwargs):
+    if not settings.DEMO_MODE:
+        return
+
+    # delete everything older than a day
+    cutoff_time = timezone.now() - timedelta(days=1)
+
+    # first mark orgs for deletion
+    org_list = Organization.objects.filter(
+        date_added__lte=cutoff_time, flags=F("flags").bitor(Organization.flags["demo_mode"])
+    )
+    org_list.update(status=OrganizationStatus.PENDING_DELETION)
+
+    # next delete the users
+    User.objects.filter(
+        date_joined__lte=cutoff_time, flags=F("flags").bitor(User.flags["demo_mode"])
+    ).delete()
```

Comment:
We should probably mark these users as `is_active=False` rather than deleting.

Reply:
@wedamija what's the advantage of doing that?

Reply:
That's just our general way of deleting users. It might be fine to straight up delete them, I mostly just don't know if there are side effects that will interfere with the org deletion or other parts of the system. We have a script that eventually does hard delete them, but I think it's only used in getsentry.

Reply:
@wedamija from what I can tell, it doesn't do bad things with org deletion. I just want to clear out old rows so that we keep the table small and avoid making this query too slow. Do you think it would make sense to first disable and then delete the next time the job runs?

Reply:
It's probably fine to just delete... I guess worst case is something goes wrong and we just make a fresh demo instance database, not like this will affect anyone using this in production.

Reply:
@wedamija So I tried setting `is_active=False` but it ended up causing my user to get into a weird state:

![Screen Shot 2021-03-04 at 3 11 05 PM](https://user-images.githubusercontent.com/8533851/110043227-e4ab5000-7cfb-11eb-865b-6e6382a22052.png)

Just get a bunch of redirects:
```
23:11:02 server  | 23:11:02 [INFO] sentry.web.frontend.base: User is not a member of any organizations
23:11:02 server  | GET 302 /auth/login/ HTTP/1.1 720
23:11:02 server  | GET 302 /organizations/new/ HTTP/1.1 683
```